.travis.yml

```diff
@@ -38,7 +38,8 @@ install:
         pip install . -r requirements.txt
       fi
 script:
-  - ./.travis-data.sh mnist binarized_mnist cifar10 cifar100
+  - pip install . -r requirements.txt
```

Comment:
We can safely remove this line: lines 31 and 37 handle installing the requirements.